src/modifiers/distribute_along_even.gd

```diff
@@ -15,31 +18,50 @@ func _init() -> void:
 func _process_transforms(transforms, _seed) -> void:
 	var path = transforms.path
 	var length: float = path.curve.get_baked_length()
-	var total_count: int = round(length / interval)
-
+	var total_count: int = round(length / interval)# + int(fmod(length, interval) <= interval / 2)
```

Comment:
What was the initial purpose of the commented formula?

Reply:
In the placement on the path, sometimes you will have enough room for another object at the end of the path, but the count would be that minus 1, so I experimented, but it introduced bugs and I was getting distracted.